histomicstk/deeplab/utils/wsi_dataset_util_large_image.py

```diff
@@ -20,6 +20,22 @@
 ################################ main function #################################
 ################################################################################
 
+def get_image(filename):
+    """
+    Read the inut image and perform channel decomposition if necessary.
+    """
+
+    slide = large_image.open(filename)
+    
+    if slide.frames == 3 and slide.bandCount == 1:
+        slide = large_image.open(filename, style={'bands': [{'framedelta': 0, 'palette': '#f00'},{'framedelta': 1, 'palette': '#0f0'},{'framedelta': 2, 'palette': '#00f'}]})
+    elif slide.frames == 1 and slide.bandCount == 3:

```

Comment:
I would allow bandCount of 3 or 4, since some images will report as having an alpha channel (which we would later ignore).